### PR TITLE
Fix missing SigningContext import in sign example

### PR DESCRIPTION
**Pull request by mayaCostantini**
#### Summary

A really small fix to include the `SigningContext` import in the example given in `sign.py`.

#### Release Note

NONE

### CODE REVIEW

File: sigstore/sign.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 ```python
 from pathlib import Path
 
-from sigstore.sign import Signer
+from sigstore.sign import Signer, SigningContext
```
Comment (woodruffw) [inline sigstore/sign.py:23]: Nit: that means this import is no longer necessary, I believe 🙂 

```suggestion
from sigstore.sign import SigningContext
```

**Review by woodruffw — APPROVED** [review]
LGTM, thank you!

**Review by woodruffw — APPROVED** [review]

**Review by tetsuo-cpp — APPROVED** [review]

**Pull request merged**